index.html

```diff
@@ -0,0 +1,138 @@
+                <footer>
+                    <p>Game designers frequently emphasize the "game" part of their title.
+                        In this inspirational talk, Stone will focus on the "designer" aspect.
+                        How is game design similar to other forms of design, such as
+                        fashion design, automotive design or industrial design? What can
+                        we learn from these other disciplines that will help us grow as game
+                        designers, both personally and professionally? Stone describes his
+                        own personal design journey and the lessons he has learned along
+                        the way. Topics will include design history, the lives of famous
+                        designers, techniques taught in design schools, and the
+                        philosophies of world-class design studios.
+                    </p>
+                </footer>
+            </li>
+
+
+            <li class="speakers">
+                <div class="author-company">
+                    <img src="https://cdn.glitch.com/46be344e-bd86-488e-bf75-ab4ed5228fee%2Fjennifer-scheurle.1024x1024.png?v=1608230528419"
+                        alt="Jennifer Scheurle" class="author-img">
+                    <div class="speaker-title">
+                        <h4>Jennifer Scheurle</h4>
+                        <h5>A multi-award-winning game designer most known the Earthlight franchise.
+                            <h4> <a href="https://www.gdcvault.com/play/1024959/Good-Game-Design-is-like">Good Game
+                                    Design is like a Magic Trick</a></h4>
+                        </h5>
+                    </div>
+                </div>
+                <footer>
+                    <p> "Tell me about some brilliant mechanics in games that are hidden
+                        from the player to get across a certain feeling." This is the question
+                        Jennifer Scheurle posed on Twitter that opened up a can of worms
+                        among the games community, as thousands of likes and replies
+                        flooded into her feed. It was a question waiting to be asked with
+                        many well-established developers sharing their secret techniques. In
+                        this talk, Jennifer will give context to these techniques that game
+                        developers have used for decades, and celebrate them with a
+                        detailed collection and how-to of those techniques from the best
+                        games and designers all over the world.
+                    </p>
+                </footer>
+            </li>
+        </ul>
+        <footer class="register">
+            <form action="">
+                <label for="email">Enter You Email to Register!</label><br /><br />
+                <input type="text" id="email" name="email" value="" /><br /><br />
```

Comment:
There is a separate `input` type just for email 👍🏻 
```suggestion
                <input type="email" id="email" name="email" value="" /><br /><br />
```